http/vibe/http/websockets.d

```diff
@@ -567,8 +680,18 @@ final class WebSocket {
 	*/
 	void close(short code = 0, scope const(char)[] reason = "")
 	{
+		//assume the default is normal, intentional closure
+		if(code == 0)
+			code = WebSocketCloseReason.normalClosure;
+
+		if(reason is null || reason.length == 0)
```

Comment:
Nitpick: the `reason is null` is redundant here, `if (reason.length == 0)` would be sufficient.